release/_git.sh

```diff
@@ -14,6 +14,17 @@ function clean_portal_repository {
 	git reset --hard && git clean -dfx
 }
 
+function clean_repository {
```

Comment:
@kiwm, can you please reuse the already existent function named `clean_portal_repository`?